tests/egl/gles2_help.c

```diff
@@ -77,15 +77,26 @@ static const char *vertexShaderText_rgba =
   "   v_texcoord  = texcoord;\n"
   "}\n";
 
+static PFNGLEGLIMAGETARGETTEXTURE2DOESPROC imageTargetTexture2DProc = NULL;
+
+void imageTargetTexture2D(EGLenum target, EGLImageKHR image)
+{
+    if (!imageTargetTexture2DProc) {
+        imageTargetTexture2DProc =
+            (void *) eglGetProcAddress("glEGLImageTargetTexture2DOES");
+    }
+    imageTargetTexture2DProc(target, image);
+}
+
```

Comment:
imageTargetTexture2D() is called multiple times and hence eglGetProcAddress(). But, imageTarget2DProc has a static pointer. Hence after the first time, its a matter of a context switch which must not be that expensive.
This is wrt "tests" folder. Why to include eglGetProcAddress() in init section?